RELEASE-NOTES.md

```diff
@@ -1,5 +1,28 @@
 # Prefect Release Notes
 
+## Release 2.19.8
+
+### Enhancements
+- Allow loading flows from an entrypoint when dependencies are missing — https://github.com/PrefectHQ/prefect/pull/14548
+
+### Fixes
+- Handle join edge case in `ParameterTypeError.from_validation_error` — https://github.com/PrefectHQ/prefect/pull/14420
+- Respect `--ui/--no-ui` flag on `prefect server start` — https://github.com/PrefectHQ/prefect/pull/14520
+- Append task run futures only when entering task run engine from flow run context — https://github.com/PrefectHQ/prefect/pull/14439
+
+## New Contributors
+* @oaustegard made their first contribution in https://github.com/PrefectHQ/prefect/pull/14505
```

Comment:
Think generally the new contributors go below all the PRs

Reply:
👍 This is how the script output them, I'll update it too.

Reply:
Looks difficult to change the script, I'll make an issue for it.